doc/whats-new.md

```diff
@@ -1,356 +1,197 @@
-Release notes for the OpenQuake Engine, version 2.2
+Release notes for the OpenQuake Engine, version 2.3
 ===================================================
 
-This release has focused on memory and performance optimizations, especially
-in the event based and scenario calculators. Moreover we have significantly
-improved our exports, by providing more information (especially for the
-event based ruptures) and/or using more efficient export formats, like the
-.npz format. As of now, the XML exports are deprecated, even if there is
-no plan to remove them at the moment.
+This release introduces several new features and improvements of the
+engine calculators. Moreover, our packaging strategy has been
+revolutionized and now we provide packages for nearly every operating
+system - including old ones like Ubuntu 12.04, previously abandoned - by
+using the exact same versions of the libraries on every supported
+platform (Linux, Windows, Mac).
```

Comment:
https://github.com/gem/oq-engine/blob/b6111109edac1caecf19715f58b64e8f06c19c84/doc/faq.md#supported-operating-systems which will become `faq.md#supported-operating-systems`

I would like do not duplicate the detailed lists of OSes (like windows 7,8,8.1,10 macos 10.10,10.11 and counting)